srml/aura/src/lib.rs

```diff
@@ -179,15 +179,19 @@ impl AuraReport {
 
 		// the number of times everyone was skipped.
 		let skipped_all = self.skipped / validator_count;
+
 		// the number of validators who were skipped once after that.
 		let skipped_after = self.skipped % validator_count;
 
 		let iter = (start_slot..validator_count).into_iter()
 			.chain(0..start_slot)
 			.enumerate();
 
+		// Don't include skipped_all in the bad block count, since that would imply the
+		// network as a whole was down, indicating a systematic issue rather than a
+		// minority of bad actors.
 		for (rel_index, actual_index) in iter {
-			let slash_count = skipped_all + if rel_index < skipped_after {
+			let slash_count = if rel_index < skipped_after {
```

Comment:
applying the above logic, I don't think we should slash at all if everyone is going to be slashed. Clearly there has been some kind of network malfunction, so I don't think slashing those who were offline a little more is helpful either.